.github/workflows/coverage.yml

```
@@ -0,0 +1,109 @@
+
+      - name: Check if any non-front-end files changed
+        id: back-end-check
+        run: |
+          changed_files=`echo "${{steps.filter.outputs.backend_files}}" | tr " " "\n" | grep -v 'specifyweb/frontend/' || echo ""`
+          echo "Changed back-end files: ${changed_files}"
+          echo "changed=$([[ -z "${changed_files}" ]] && echo "" || echo "1")" >> $GITHUB_OUTPUT
+
+  setup:
+    needs: changes
+    if: |
+      (needs.changes.outputs.backend_changed  ||
```

actionlint (1.7.12)

[error] property "frontend_changed" is not defined in object type {backend_changed: string; frontend: string; frontend_changes: string}

(expression)